src/site/content/en/metrics/inp/index.md

```diff
@@ -54,7 +54,7 @@ INP is a metric that assesses a page's overall responsiveness to user interactio
 A note on how INP is calculated
 {% endDetailsSummary %}
 
-As stated above, INP is calculated by observing all the interactions made with a page. The chosen value is then a [percentile](https://en.wikipedia.org/wiki/Percentile) of those interactions. A formula is then used to choose a high percentile value of those interactions. For pages with few interactions, the interaction with the worst latency (the 100th percentile) is chosen. For pages with many interactions, the 99th or 98th percentile is chosen.
+As stated above, INP is calculated by observing all the interactions made with a page. The resulting value is determined by a formula that calculates a high [percentile](https://en.wikipedia.org/wiki/Percentile) value of all the observed interactions. The interaction with the worst latency is chosen, dropping the worst of every 50 interactions.
```

Comment:
Would be awesome if we just didn't say "percentile" at all. cc @mmocny for thoughts.

Reply:
Agreed!

I think if you just drop the line completely and leave the last one it still works.  However, maybe we should expand what we mean by dropping. Perhaps something like:

"The interaction with the worst latency is reported as INP, however, one worst interaction is ignored for every 50 interactions.  The vast majority of page experiences do not have over 50 interactions."